From 152f5a590ad349922cc90e3e867a599eced7fada Mon Sep 17 00:00:00 2001
From: Karl Williamson <khw@cpan.org>
Date: Sun, 30 Sep 2018 10:33:22 -0600
Subject: [PATCH] regexec.c: Rename variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The new name is clearer as to its meaning, more so after the next
commit.

Petr Písař: Ported to 5.28.1 from
81ec018c6daca2b4c8c87eb335a371b4c90753f3.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 regexec.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/regexec.c b/regexec.c
index d1a3937..899d979 100644
--- a/regexec.c
+++ b/regexec.c
@@ -10633,20 +10633,20 @@ Perl_isSCRIPT_RUN(pTHX_ const U8 * s, const U8 * send, const bool utf8_target)
                 || zero_of_run == '0'
                 || script_zeros[script_of_char] == 0))
         {
-            SSize_t range_zero_index;
-            range_zero_index = _invlist_search(decimals_invlist, cp);
-            if (   LIKELY(range_zero_index >= 0)
-                && ELEMENT_RANGE_MATCHES_INVLIST(range_zero_index))
+            SSize_t zero_of_char_index;
+            zero_of_char_index = _invlist_search(decimals_invlist, cp);
+            if (   LIKELY(zero_of_char_index >= 0)
+                && ELEMENT_RANGE_MATCHES_INVLIST(zero_of_char_index))
             {
-                UV range_zero = decimals_array[range_zero_index];
+                UV zero_of_char = decimals_array[zero_of_char_index];
                 if (zero_of_run) {
-                    if (zero_of_run != range_zero) {
+                    if (zero_of_run != zero_of_char) {
                         retval = FALSE;
                         break;
                     }
                 }
                 else {
-                    zero_of_run = range_zero;
+                    zero_of_run = zero_of_char;
                 }
             }
         }
-- 
2.17.2

